services/dis-console-ui/Makefile

```
@@ -0,0 +1,198 @@
+	$(BUN) run dev
+
+.PHONY: dev-podman
+dev-podman: ## Start the Vite dev server in a Bun container (no host toolchain needed). Serves http://localhost:5173.
+	$(CONTAINER_TOOL) run --rm -it \
+		-v "$(CURDIR):/app" -v $(NM_VOLUME):/app/node_modules -w /app \
+		-p 5173:5173 -e CHOKIDAR_USEPOLLING=true \
+		$(BUN_IMAGE) \
+		sh -lc "bun install && bun run dev -- --host 0.0.0.0"
+
+.PHONY: dev-live
+dev-live: ## Like dev-podman but against a live fleet API (BACKEND=a port-forwarded server). Proxies /api, so no CORS.
```

checkmake (0.3.2)

[warning] Target body for "dev-live" exceeds allowed length of 5 lines (6).

(maxbodylength)